### ISSUE TITLE
Add Top Stories to Secondary Column

### ISSUE THREAD

**Issue by simonsinclair** [bbc/simorgh#5700]
### Is your feature request related to a problem? Please describe.
Add Top Stories to the Secondary Column, following the merge of https://github.com/bbc/simorgh/pull/5417. Top Stories will be rendered using a similar pattern to CPS Related Content, where `content` below is Top Stories data:

```jsx
<StoryPromoUl>
  {content
    .map(item => (
      <StoryPromoLi key={item.id || item.uri}>
        <StoryPromo item={item} />
      </StoryPromoLi>
    ))}
</StoryPromoUl>;

```

### Describe the solution you'd like
1. The Top Stories end-point is the front page. The data comes from the CPS group 'Top Story' or 'Top Stories', which is held on the front page. Extract fixture data for Top Stories from 

- https://ares-broker.api.bbci.co.uk/api/index/pidgin/front_page
- https://ares-broker.api.bbci.co.uk/api/index/mundo/front_page

to be used in the component, till the Ares endpoint is ready.

for Pidgin, the CPS group name on the front page  is 'Top Stories'
```
content: {
groups: [
{
type: "responsive-top-stories",
title: "Top Stories",
```

for Mundo, the CPS group name on the front page is 'Top Story'
```
content: {
groups: [
{
type: "top-story",
title: "Top story",
```
2. Use `psammead-story-promo-list` in a similar way shown in the example above to render the dummy fixture data in a secondary column component container.
3. Use `psammead-section-label` to create a "Top Stories" section label from the fixture value created in step 1.
4. A11y - This should be wrapped in a section html element with a role region attribute which is labelled via an aria-labelledby attribute which maps to the h2. Below is an example from a front page section

<img width="628" alt="Screen Shot 2020-03-02 at 11 56 12" src="https://user-images.githubusercontent.com/1061401/75674524-22a30d80-5c7d-11ea-94f6-a0c1a8edd471.png">

5. Review how this looks with UX
6. This needs an a11y swarm.

### Describe alternatives you've considered
n/a

### Testing notes
- Given I use a screen reader
- When I navigate to the top stories section on an STY page
- Then a region landmark must be announced e.g. Region
- And this must be announced with a label 
- And the label must be the same as the h2 (e.g. Top stories) and be unique to the page
- And this should not be inside the main landmark or any other landmark

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

### Additional context
- https://paper.dropbox.com/doc/Simorgh-WS-STY-onward-journeys--AvIEmS7B0c8VTK8ym0qb6uTrAg-gSSePsYXjWttnHdEIwWYI
- The component header is also taken from the front page data. See #5794. The CPS group has a semantic group name with a translation. E.g. for Pidgin (https://ares-broker.api.bbci.co.uk/api/index/pidgin/front_page) the CPS group for Top Stories is 'Top Stories'. This group has a group name of "Top tori", so the heading for the Top Story onward journey on the STY page is "Top tori"
```js
strapline: {
  name: "Top tori"
},
semanticGroupName: "Top Stories"
```

**Comment by simonsinclair** [comment]
Unblocked following the merge of https://github.com/bbc/simorgh/pull/5417.
